### PR TITLE
[DRAFT] EECS 149 Lab Applications

### PR DESCRIPTION
**Pull request by gundralaa**
LF adaptions of EECS 149 lab robot state machines for the buckler+nrf+romi platform.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,35 @@
 
 This repo provides resources for using Lingua Franca to program an nRF52x Thread/BLE embedded board from Nordic Semiconductor with a Berkeley Buckler daughter card.
 
+## VM Setup Instructions
+This set of instructions will guide a user on how to setup the EECS149 Lab VM to be able to use Lingua Franca. The installation requires sudo permissions on the vm.
```
Comment (edwardalee) [inline README.md:6]: Perhaps include a link to the Lab VM here?

File: README.md [hunk 2]
```diff
@@ -11,7 +40,7 @@ git submodule update --init --recursive
 
 ## Install Cross-Compilation and Loading Tools
```
Comment (edwardalee) [inline README.md:41]:
```suggestion
## Install Cross-Compilation and Loading Tools (Skip if using the Lab VM)
```

File: README.md (hunk 1 above)
Comment (edwardalee) [inline README.md:6]:
```suggestion
The easiest way to set up is to use the EECS 149 lab virtual machine.
This set of instructions will guide a user on how to setup the EECS149 Lab VM to be able to use Lingua Franca. The installation requires sudo permissions on the vm.
```

**Review by edwardalee — CHANGES_REQUESTED** [review]
There is a problem with this PR. It shows every line of every file changed. I suspect this is a platform misconfiguration where, for example, the newline character has been changed. Git normally has no difficulty with variants of newline on different platforms, so this seems like a misconfiguration.

As I consequence, I cannot see what has changed.

**Comment by lhstrh** [comment]
Closing this PR to avoid confusion as it has been subsumed by https://github.com/icyphy/lf-buckler/pull/2.

**Pull request closed without merging**